typescript/src/e2e.test.ts

```diff
@@ -1542,6 +1543,1054 @@ describe("CDP Client E2E Tests", () => {
     });
   });
 
+  describe("Policy Evaluation", () => {
+    let policy: Policy;
+
+    beforeAll(async () => {
+      policy = await cdp.policies.createPolicy({
+        policy: {
+          description: Date.now().toString(),
+          scope: "account",
+          rules: [
+            {
+              action: "accept",
+              operation: "signEvmMessage",
+              criteria: [{ match: "^$", type: "evmMessage" }],
+            },
+          ],
+        },
+      });
+    });
+
+    async function cleanup() {
+      await cdp.evm.updateAccount({
+        address: testAccount.address,
+        update: {
+          accountPolicy: "",
+        },
+      });
+      await cdp.policies.deletePolicy({
+        id: policy!.id,
```

Comment:
nitty nit: could add if statement to check if policy is there so that we dont use `!`